src/components/views/TitleFormInfo.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+import PropTypes from 'prop-types';
+
+import {
+  Checkbox,
+  Col,
+} from '@folio/stripes/components';
+
+const TitleFormInfo = ({
+  isSuppressFromDiscoveryEnabled,
+  name,
+}) => (
+  <div data-test-platform-form-info>
+    <div data-test-title-instance-name>
+      {name}
+    </div>
+    { isSuppressFromDiscoveryEnabled('title') ?
+      <Col xs={3}>
+        <Field
```

Comment:
The checkbox needs to be left aligned

Reply:
It is left aligned per default, isn't it?